units/en/unitbonus1/how-huggy-works.mdx

```diff
@@ -0,0 +1,66 @@
+# How Huggy works? [[how-huggy-works]]
+
+Huggy is a Deep Reinforcement Learning environment made by Hugging Face and based on [Puppo the Corgi, a project by the Unity MLAgents team](https://blog.unity.com/technology/puppo-the-corgi-cuteness-overload-with-the-unity-ml-agents-toolkit).
+This environment was created using the [Unity game engine](https://unity.com/) and [MLAgents](https://github.com/Unity-Technologies/ml-agents). ML-Agents is a toolkit for the game engine Unity that allows us to **create environments using Unity or use pre-made environments to train our agents**.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/unit-bonus1/huggy.jpg" alt="Huggy" width="100%">
+
+So, in this environment, we aim to train Huggy to **fetch the stick we throw at him. It means he needs to move correctly toward the stick**.
+
+## The State Space: what Huggy "perceives." [[state-space]]
+Huggy doesn't "see" his environment. Instead, we provide him information about the environment:
+The target (stick) position
+The relative position between himself and the target
+The orientation of his legs.
+Given all this information, Huggy can decide which action to take next to fulfill his goal.
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/notebooks/unit-bonus1/huggy.jpg" alt="Huggy" width="100%">
```

Comment:
This same image was shown slightly above, should we remove this one?

Reply:
:heavy_check_mark: